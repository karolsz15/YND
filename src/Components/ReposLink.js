import React from 'react';
import classes from '../Container/Container.module.css';

const ReposLink = props => (
    <a className={classes.Wrap} href={`https://github.com/${props.username}`} target="_blank" rel="noopener noreferrer">
        <div className={classes.SingleRepo}>
            <div className={classes.TitleAndDescription}>
                <div className={classes.RepoTitle}>
                    All {props.username}'s repositories...
                    <i class="fa fa-arrow-right" aria-hidden="true"></i> 
                </div>
                <div className={classes.RepoDescription}>
                    Click here to see all {props.username}'s repositories!
                </div>
            </div>
        </div>
    </a>
);

export default ReposLink;