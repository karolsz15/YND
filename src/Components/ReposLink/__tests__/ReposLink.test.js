import React from 'react';
import { shallow } from 'enzyme';
import ReposLink from '../ReposLink';

const wrapper = shallow(<ReposLink />);

describe('ReposLink component', () => {
  it('should render the ReposLink correctly', () => {
    expect(wrapper).toMatchSnapshot();
  });
});