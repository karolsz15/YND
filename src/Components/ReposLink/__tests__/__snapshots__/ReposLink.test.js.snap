// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ReposLink component should render the ReposLink correctly 1`] = `
<a
  className="Wrap"
  href="https://github.com/undefined"
  rel="noopener noreferrer"
  target="_blank"
>
  <div
    className="SingleRepo"
  >
    <div
      className="TitleAndDescription"
    >
      <div
        className="RepoTitle"
      >
        All 
        's repositories...
        <i
          aria-hidden="true"
          className="fa fa-arrow-right"
        />
      </div>
      <div
        className="RepoDescription"
      >
        Click here to see all 
        's repositories!
      </div>
    </div>
  </div>
</a>
`;
